tailwind.config.js

```diff
@@ -36,31 +36,28 @@ export default {
                     text: color('--secondary-text', colors.white),
                 },
 
+                conversion: {
+                    DEFAULT: color('--conversion', colors.green[500]),
+                    text: color('--conversion-text', colors.white),
+                },
+
                 foreground: {
-                    emphasis: color('--foreground-emphasis', colors.gray[800]),
-                    DEFAULT: color('--foreground', colors.gray[700]),
-                    muted: color('--foreground-muted', colors.gray[500]),
+                    emphasis: color('--foreground-emphasis', colors.slate[900]),
+                    DEFAULT: color('--foreground', colors.slate[800]),
+                    muted: color('--foreground-muted', colors.slate[600]),
                 },
 
                 border: {
-                    emphasis: color('--border-emphasis', colors.gray[500]),
-                    DEFAULT: color('--border', colors.gray[200]),
-                    muted: color('--border-muted', colors.gray[100]),
+                    emphasis: color('--border-emphasis', colors.slate[500]),
+                    DEFAULT: color('--border', colors.slate[200]),
+                    muted: color('--border-muted', colors.slate[100]),
                 },
 
                 background: {
-                    emphasis: color('--background-emphasis', colors.gray[200]),
-                    DEFAULT: color('--background', colors.gray[100]),
-                    muted: color('--background-muted', colors.gray[50]),
+                    emphasis: color('--background-emphasis', colors.slate[900]),
+                    DEFAULT: color('--background', colors.slate[800]),
+                    muted: color('--background-muted', colors.slate[50]),
```

Comment:
In our designs we have 3 variants of each color type.

so since white is the most used background color, that should be default just like primer does.
We often use the text color which is slate 800 in this case, as a background color so i think it should be a big step. And muted should be a lighter version than default, but that's not really the case in this example.

```
emphasis: color('--background-emphasis', colors.slate[800]),
DEFAULT: color('--background', colors.white),
muted: color('--background-muted', colors.slate[50]),
```